boefjes/boefjes/job_handler.py

```diff
@@ -83,7 +83,7 @@ def get_environment_settings(boefje_meta: BoefjeMeta, environment_keys: List[str
             f"{settings.katalogus_api}/v1/organisations/{boefje_meta.organization}/{boefje_meta.boefje.id}/settings"
         ).json()
 
-        return {k: v for k, v in environment.items() if k in environment_keys}
+        return {k: str(v) for k, v in environment.items() if k in environment_keys}
     except RequestException:
         logger.exception("Error getting environment settings")
```

Comment:
It returns an empty dict below. But I think it wouldn't hurt to have another look at the environment/settings setup, since we can also check wether the boefje defines a schema and skip this call altogether if it doesn't. Then we could crash when the API call fails.

Also in the current setup where these handlers run in the same context as the Katalogus, we do not need to make API calls but can simply query the settings from the repository itself. This would also reduce the chance of crashing since we bypass an extra API call.

Reply:
I did the check and reraising, not bypassing the API (yet).